public/app/features/scopes/selector/ScopesSelectorService.ts

```diff
@@ -1,4 +1,4 @@
-import { isEqual } from 'lodash';
+import { isEqual, last } from 'lodash';
```

Comment:
This file is mostly just added comment and small refactor. Thought the URL handling would be here but in the end moved it to scopesService but wanted to keep the updates.